src/sw.ts

```diff
@@ -3,11 +3,55 @@
 
 import {getFullBaseUrl} from './helpers/getFullBaseUrl'
 
+import {useTaskStore} from '@/stores/tasks'
+import {useAuthStore} from '@/stores/auth'
+
 declare let self: ServiceWorkerGlobalScope
 
 const fullBaseUrl = getFullBaseUrl()
+const authStore = useAuthStore()
```

Comment:
Already when only including a reference to either the authstore or taskstore, the build will fail